### DISCUSSION TITLE
[Desktop][Bug] GitHub homepage looks weird.

### DISCUSSION THREAD

**Discussion by Hephaest** [community/community#2953] [Discussions] [answered]
It has been a while that I didn't visit the homepage, but it is good to see some new intros/features.

However, I find that it has 2 issues:

1. Pixel distortion
2. Crossed UI component (As highlighted below)

<p align="center">
    <img src="https://user-images.githubusercontent.com/37981444/101856397-28c90180-3ba0-11eb-940a-43077be20093.jpg">
 </p>

In fact, this **earth** is too big. The smaller version could be more suitable.

**Comment by alexfinnarn** [comment]
This earth is neither too big nor too small, it just is. The smaller version would be more appropriate in the grand scheme of things, though.

**Comment by becca** [comment, marked as answer]
This space is currently reserved for feedback on the Discussions and GitHub for mobile features.
